src/wp-admin/site-editor.php

```diff
@@ -59,7 +59,17 @@ static function( $classes ) {
 	'/wp/v2/global-styles/' . $active_global_styles_id . '?context=edit',
 	'/wp/v2/global-styles/' . $active_global_styles_id,
 	'/wp/v2/themes/' . $active_theme . '/global-styles',
+	'/__experimental/block-navigation-areas?context=edit',
 );
+
+$areas        = get_option( 'fse_navigation_areas', array() );
+$active_areas = array_intersect_key( $areas, get_navigation_areas() );
+foreach ( $active_areas as $post_id ) {
+	if ( $post_id ) {
+		$preload_paths[] = "/wp/v2/navigation/$post_id?context=edit";
```

Comment:
```suggestion
		$preload_paths[] = add_query_args('context', 'edit', rest_get_route_for_post( $post_id ) );
```

Reply:
We made this change else where in core. For post and term endpoints, let's use function going forward.